dragonchain/exceptions.py

```diff
@@ -151,11 +151,11 @@ class SanityCheckFailure(DragonchainException):
     """Exception raised when sanity check fails"""
 
 
-class RPCError(DragonchainException):
-    """Exception raise when RPC has an error"""
+class InterchainConnectionError(DragonchainException):
+    """Exception raise when RPC / API call has an error"""
 
 
-class RPCTransactionNotFound(DragonchainException):
+class TransactionNotFound(DragonchainException):
```

Comment:
This is thrown specifically for interchain transactions, so I would be more specific. This isn't clear if it's a dragonchain or interchain transaction